src/main/fulcro/client.cljc

```diff
@@ -73,7 +73,20 @@
 ; easier to access, so the user don't have to require a impl namespace to reference
 (def mutate app/write-entry-point)
 
-(defn new-fulcro-client
+
+(s/def ::load-marker-default (s/or :key keyword? :falsey (s/or :nil nil? :false false?)))
```

Comment:
So, I'd drop the specs, and this validate function.  Let's let them set it to whatever the heck they want...it doesn't really matter (your spec isn't even technically right, since load marker names can be anything now). Drop the validate function...just make the "default if unspecified" be false.